sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -490,6 +518,7 @@ public Job getJob(JobReference jobRef, Sleeper sleeper, BackOff backoff)
     private DatasetServiceImpl(BigQueryOptions bqOptions) {
       this.errorExtractor = new ApiErrorExtractor();
       this.client = newBigQueryClient(bqOptions).build();
+      this.newWriteClient = newBigQueryWriteClient(bqOptions);
```

Comment:
We're supposed to manage the lifecycle of this client and close it when done. Since this code doesn't do that users see `SEVERE` errors (e.g. BEAM-12359) whenever they use BigQueryIO. I field BEAM-12365 for this.